src/platform/forms-system/src/js/components/FormNav.jsx

```diff
@@ -70,11 +71,17 @@ export default class FormNav extends React.Component {
             aria-valuemax={chapters.length}
             className="nav-header nav-header-schemaform"
           >
-            <h2 className="vads-u-font-size--h4">
-              <span className="form-process-step current">{current}</span>{' '}
-              <span className="form-process-total">of {chapters.length}</span>{' '}
-              {chapterName}
-            </h2>
+            {environment.isDev() ? (
+              <h2 className="vads-u-font-size--h4">
+                {`${current} of ${chapters.length} ${chapterName}`}
```

Comment:
We need to test that removing the spans from the h2 fixes the iOS problem referenced in the linked issue.